rest/adminapitest/collections_admin_api_test.go

```diff
@@ -75,12 +75,6 @@ func TestCollectionsSyncImportFunctions(t *testing.T) {
 	resp := rt.SendAdminRequest(http.MethodPut, "/db/", bucketConfig)
 	rest.RequireStatus(t, resp, http.StatusCreated)
 
-	resp = rt.SendAdminRequest(http.MethodGet, "/db/_config/import_filter", "")
-	rest.RequireStatus(t, resp, http.StatusBadRequest)
-
-	resp = rt.SendAdminRequest(http.MethodGet, "/db/_config/sync", "")
-	rest.RequireStatus(t, resp, http.StatusBadRequest)
-
```

Comment:
There's no longer a /db/ becomes ambiguous in a multiple collection context. This ends up returning alternately 404 or 400, I am going to fix this in a followup ticket.

Reply:
Furthermore, this is confusing in the case where we are using the default collection because it is actually a different case than the non default collection. I didn't think it was worth testing this use case.

Reply:
ah ok, I hadn't noticed that we already have keyspace1/keyspace2 tests for these endpoints just below.  I think this makes sense.